### PR TITLE
Feat: index the daily user debt

### PR DESCRIPTION
**Pull request by shardAstronaut**
### Description

- Modified the logic to store daily the users debt if it was updated.
- Updated the logic to clean the database.
- Notion [link](https://www.notion.so/spaceshard/Team-overview-daa7327f21e84c31b96647cec9603d26?p=75dbb430dde3484cbad672ce5b43a18e&pm=s)
### Code review notes

- Make use the daily user debt logic is good and the data is stored correctly.

### Checklist:

- [X]  Checked the changes locally.
- [ ]  Created / updated tests (e2e / unit).

### CODE REVIEW

File: src/nimbora-liquity/liquity.service.ts [hunk 1]
```diff
@@ -73,32 +73,85 @@ export class LiquityService implements Service {
...
+          return;
+        } else if (from == troves[0].contract || from == troves[1].contract) {
+          // Borrow the `to` is the user and the `from` is the trove address.
+          const totalDebtId = `${to}_${from}`;
+          const dailyDebtId = `${to}_${from}_${timestamp}`;
+
+          // Increase debt by 1% to include liquity fee when borrow.
+          // amount = amount + amount / 100n;
+
+          debt = await Liquity_Debt.loadEntity(totalDebtId);
```
Comment (ioanSL) [inline src/nimbora-liquity/liquity.service.ts:103]: When loading the debt entities, it gets repetitive. Take into account that we not only add data to the DB, but also update existing fields, can you add a function called `loadDebtEntities` with one argument `id` that returns the debt entity or creates a new one if it doesn't exist.

**Review by ioanSL — CHANGES_REQUESTED** [review]
When loading the debt entities, it gets repetitive. Taking into account that we not only add data to the DB, but also update existing fields, can you add a function called `loadDebtEntities` with one argument `id` that returns the debt entity or creates a new one if it doesn't exist.

**Comment by shardAstronaut** [comment]
> When loading the debt entities, it gets repetitive. Taking into account that we not only add data to the DB, but also update existing fields, can you add a function called `loadDebtEntities` with one argument `id` that returns the debt entity or creates a new one if it doesn't exist.

It should be fine to keep it as it doesn't affect the core logic, we can refactor the code later if needed.

**Review by ioanSL — APPROVED** [review]
LGTM 👍🏽

**Pull request merged**